src/snapred/backend/recipe/algorithm/RemoveEventBackground.py

```diff
@@ -64,18 +74,14 @@ def chopIngredients(self, predictedPeaksList: List[GroupPeakList]):
             self.maskRegions[peakList.groupID] = []
             for peak in peakList.peaks:
                 self.maskRegions[peakList.groupID].append((peak.minimum, peak.maximum))
-
         # get handle to group focusing workspace and retrieve all detector IDs in each group
         focusWSname: str = str(self.getPropertyValue("GroupingWorkspace"))
-        focusWS = self.mantidSnapper.mtd[focusWSname]
-        self.groupIDs: List[int] = [int(x) for x in focusWS.getGroupIDs()]
+        # get a list of the detector IDs in each group
+        self.groupDetectorIDs = access_pointer(GroupedDetectorIDs(focusWSname))
```

Comment:
This line confuses me, we are accessing the result of GroupedDetectorIDs?  I understand its output is a pointer (after looking up the Algo) but it feels a bit obtuse to have to unwrap it via some other functional import
Pointers are also not common enough in our code for this pattern to be apparent yet.

Could you perhaps do something like this to make it more obvious:
```
from mantid.kernel import ULongLongPropertyWithValue as PointerProperty

...
        
        result : PointerProperty = GroupedDetectorIDs(focusWSname)
        self.groupDetectorIDs = access_pointer(result)
```

Reply:
I can do this, but it's worth noting that the result is not a "PointerProperty" but an integer, so the type isn't really right... but since type hints are only just that anyway, it works okay.